### PR TITLE
Translations/portuguese

### PR DESCRIPTION
**Pull request by popperz0r**
First version of email translations.

If this goes well, ill add more email variables to translate.

### CODE REVIEW

File: config/locales/pt.yml [hunk 1]
```diff
@@ -191,13 +191,33 @@ pt:
   create_new_community: "Criar uma Nova Comunidade"
 
   # Mail messages
-  hello: "Olá %{user_name}"
-  new_user: "Obrigado pelo seu contacto. A sua mensagem foi recebida pela nossa equipa de suporte e iremos entrar em contacto consigo o mais rapidamente possível."
-  account_generated: "Foi criada uma nova conta para si. Para fazer login, use este link:"
-  above_this_line: "Escreva a sua resposta acima desta linha"
-  message_id: "Id da Mensagem"
-  message_response: "A seguinte mensagem foi criada em resposta à sua questão:"
-  view_online: "Ver isto online:"
```
Comment (scott) [inline config/locales/pt.yml]: @popperz0r Tiago, Sorry again for the delay.  I am curious if you really meant to replace the PT translations with English (these are the strings you sent me via Twitter).

File: config/locales/pt.yml [hunk 2]
```diff
@@ -191,13 +191,33 @@ pt:
...
+      reset_link_request: "Someone has requested a link to change your password. You can do this through the link below"
+      change_password: "Change my password"
+      mail_ignore: "If you didn't request this, please ignore this email"
+      ignore_notice: "Your password won't change until you access the link above and create a new one"
+
+  #Mail Message Unlock Instructions #devise/mailer/unlock_instructions.html.erb
+  unlock_instructions:
+      account_locked: "Your account has been locked due to an excessive number of unsuccessful sign in attempts"
+      unlock_link: "Click the link below to unlock your account"
+      unlock_account: "Unlock my account"
```
Comment (scott) [inline config/locales/pt.yml]: This is great, but these are already translated, take a look in `config/locales/devise`.   That being said, I am not sure the email template files are set up to use them.

**Comment by Rynaro** [comment]
I'm portuguese speaker, and reading the translation, all lines seems good to me.
_(In another projects, like Brackets, is common another "language speaker" validates the translation, for more accuracy translation)_ 
I don't know if Helpy project has this practice.
Good work @popperz0r

**Comment by scott** [comment]
Hi @Rynaro thanks for the vote of confidence.  I like the idea of having things validated!

**Comment by CGA1123** [comment]
#319 Makes this PR obsolete.

@scott Can you close?

**Pull request closed without merging**